src/components/menu/Aidrdrop/AirdropModal.tsx

```diff
@@ -11,6 +11,48 @@ import { useUserWalletDataContext } from '../../../libs/web3-data-provider';
 import { isValid } from '../../../helpers/number';
 import { useProtocolDataContext } from '../../../libs/protocol-data-provider';
 
+const UIINCENTIVEDATAPROVIDER_ABI = [
```

Comment:
We should probably query data via `@pret/contract-helper` package. 

We need to update the package here: https://github.com/pret-labs/pret-utilities/blob/master/packages/contract-helpers/src/ui-incentive-data-provider/index.ts